docs/chapters/architecture/index.rst

```diff
@@ -10,3 +10,4 @@ The following chapters describes the architecture of the PELUX platform.
    SOTA/index.rst
    DLT/index.rst
    SWC/index.rst
+   Lifecycle/index.rst
```

Comment:
If you rebase, you will notice the difference in the structure. It will be like /docs/chapters/architecture/**subsystems/**Lifecycle/index.rst